README.md

```diff
@@ -231,6 +232,23 @@ Some of the options for your `runner` are listed below:
 
 </details>
 
+</details>
+<!-- Notes on using rp2040_hal and rp2040_boot2 -->
+<details open="open">
+  <summary><h2 style="display: inline-block" id="notes-on-using-rp2040_boot2">Notes on using rp2040_boot2</h2></summary>
+
+  When rp2040_boot2 is used instead of rp-pico the bootloader
+  must be written to the .boot2 sector. This can be done by adding the
```

Comment:
I would suggest a small change to that sentence:
```suggestion
  The second-stage boot loader must be written to the .boot2 section. That
  is usually handled by the board support package (e.g.`rp-pico`). If you don't use
  one, you should initialize the boot loader manually. This can be done by adding the
```

Reply:
This is a great suggestion, making it clear why it is needed.